infrastructure/ansible/roles/ats_sim/README.md

```diff
@@ -0,0 +1,63 @@
+<!--
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+-->
+ats_sim
+=========
+
+
+
+Requirements
+------------
+
+
+
+Role Variables
+--------------
+
+Refer to the defaults/main.yml for most information.
+
+Dependencies
+------------
+
+None
+
+Example Playbook
+----------------
+```yaml
+  - name: Deploy ats_sim
+    import_role:
+      name: ats_sim
+    vars:
+      install_ats_sim: true
+      sim_to_url: https://to.exenv.cdnlab.comcast.net
+      sim_to_user: {{ to_user }}
+      sim_to_password: {{ to_password }}
+      sim_to_cdn: Kabletown2.0
+      sim_git_repo: "https://github.com/apache/trafficcontrol.git"
+      sim_git_version: master
```

Comment:
`sim` is a probably overly generic as the ansible prefix, I'd recommend `ats_sim`.  Just in case we add other kinds of simulators someday.